python/py-keep/Portfile

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-keep
+version             2.10.1
+license             MIT
+supported_archs     noarch
+maintainers         nomaintainer
+
+description         A Meta CLI toolkit
+long_description    A Meta CLI toolkit: your personal shell command keeper
+homepage            https://pypi.org/project/keep/
+master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
+distname            keep-${version}
+checksums           rmd160  5bef8733deb9231da85b96993d381eb169533909 \
+                    sha256  3abbe445347711cecd9cbb80dab4a0777418972fc14a14e9387d0d2ae4b6adb7 \
+                    size    13044
+
+python.versions     37 38 39 310
+
+if {${name} ne ${subport}} {
+   depends_build-append port:py${python.version}-setuptools
+   depends_lib-append   port:py${python.version}-terminaltables \
+                        port:py${python.version}-requests \
+                        port:py${python.version}-click \
+                        port:py${python.version}-pygit2
```

Comment:
please do run `port lint` (in this case the best thing to do is `port lint --nitpick subportof:py-keep`, which will lint all the subports) and build the port locally - you would have noticed a missing dependency: 
```Error: Unknown dependency: py310-pygit2```

Reply:
> please do run `port lint` (in this case the best thing to do is `port lint --nitpick subportof:py-keep`, which will lint all the subports) and build the port locally - you would have noticed a missing dependency: `Error: Unknown dependency: py310-pygit2`

I have already removed py310 variant, but thanks for pointing out so quickly. (I did the --nitpick thing, but it did not complain.)

Reply:
we should not remove the PY310 variant as that is the default version in MacPorts now - why can one not add the missing subports for the dependencies?